remote-playback/event-handlers-manual.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <title>
+    Test that all event handlers are called.
+  </title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/common/media.js"></script>
+  <body>
+    <div id="prep">
+      <p>Please make sure a device for remote playback is <b>available.</b></p>
+      <button id="prompt-button-prep">Show devices</button>
```

Comment:
I would have designed this test a little differently.  Instead of requiring the user to prompt once, I would start with the "show devices" enabled and "start test" disabled.

Below, I would call watchAvailability on v.  If the callback fires with `true`, then disable "show devices" and enable "start test".  Chrome, which looks for devices in the background, should be able to start the test right away. 

If the callback fires with `false`, then the user can click "show devices" to get the browser to look for devices in the background.

If watchAvailability rejects, then there's no way to know if there are devices available or not, and you might as well enable 'start test'.

Make sense?

(The same comment applies to all tests that require a device to be available).

Reply:
When writing these tests I always try to utilize as few features of the same API as possible, so that testing individual functionality is not affected by misbehaviour of another functionality.

In this case, if the watchAvailability is not working properly and return a false `true` then this test may fail, even if the event handlers work correctly.

Does that make sense?